chart/k8gb/templates/coredns-cm.yaml

```diff
@@ -11,13 +11,12 @@ data:
         errors
         health
         ready
-        loadbalance round_robin
         prometheus 0.0.0.0:9153
         forward . /etc/resolv.conf
         k8s_crd {
-            resources DNSEndpoint
```

Comment:
@kuritka it's not not immediately clear from https://github.com/k8gb-io/coredns-crd-plugin/pull/45 - why do we need to remove this config line?

Reply:
`resources DNSEndpoint ` field was meaningless, we didn't work with it in any way.

Reply:
@kuritka cool, I was mostly wondering why it actually breaking the terratest, but cool to get rid of it any case, thanks!